#! /bin/sh /usr/share/dpatch/dpatch-run
## 42_check_linux_raid_fix_r10.dpatch by Matija Nalis <mnalis-debianbug@voyager.hr> 
##
## DP: Make Raid10+ working

@DPATCH@
diff -urNad nagios-plugins-1.4.14~/contrib/check_linux_raid.pl nagios-plugins-1.4.14/contrib/check_linux_raid.pl
--- nagios-plugins-1.4.14~/contrib/check_linux_raid.pl	2009-12-08 19:46:00.000000000 +0100
+++ nagios-plugins-1.4.14/contrib/check_linux_raid.pl	2009-12-08 19:46:31.000000000 +0100
@@ -44,7 +44,7 @@
 
 my $nextdev;
 if(defined $ARGV[0]) { $nextdev = shift; }
-else { $nextdev = "md[0-9]"; }
+else { $nextdev = "md[0-9]+"; }
 
 my $code = "UNKNOWN";
 my $msg = "";
